ledger/internal/cow_creatables.go

```diff
@@ -74,41 +74,21 @@ func (cs *roundCowState) GetAssetParams(addr basics.Address, aidx basics.AssetIn
 }
 
 func (cs *roundCowState) PutAppParams(addr basics.Address, aidx basics.AppIndex, params basics.AppParams) error {
-	// TODO: remove after the schema switch
-	if _, ok := cs.mods.NewAccts.GetData(addr); !ok {
-		return fmt.Errorf("PutAppParams: %s not found in deltas for %d", addr.String(), aidx)
-	}
-
 	cs.mods.NewAccts.UpsertAppParams(addr, aidx, &params)
 	return nil
 }
 
 func (cs *roundCowState) PutAppLocalState(addr basics.Address, aidx basics.AppIndex, state basics.AppLocalState) error {
-	// TODO: remove after the schema switch
-	if _, ok := cs.mods.NewAccts.GetData(addr); !ok {
-		return fmt.Errorf("PutAppLocalState: %s not found in deltas for %d", addr.String(), aidx)
-	}
-
 	cs.mods.NewAccts.UpsertAppLocalState(addr, aidx, &state)
 	return nil
 }
 
 func (cs *roundCowState) PutAssetHolding(addr basics.Address, aidx basics.AssetIndex, data basics.AssetHolding) error {
-	// TODO: remove after the schema switch
-	if _, ok := cs.mods.NewAccts.GetData(addr); !ok {
-		return fmt.Errorf("PutAssetHolding: %s not found in deltas for %d", addr.String(), aidx)
-	}
-
 	cs.mods.NewAccts.UpsertAssetHolding(addr, aidx, &data)
 	return nil
 }
 
 func (cs *roundCowState) PutAssetParams(addr basics.Address, aidx basics.AssetIndex, data basics.AssetParams) error {
-	// TODO: remove after the schema switch
```

Comment:
note `DeleteAssetParams` below still have such check. This is because delete is real deleting and requires `Total*s` update,  and `PutAssetParams` is for both create and update. Maybe it makes sense to separate them